lobby/README.md

```diff
@@ -22,3 +22,54 @@ This is the well-known running TripleA lobby server.
 ../gradlew jibDockerBuild
 docker run -d --name=dev-lobby -p 3304:3304 lobby:1.10.dev
 ```
+
+TODO: update below, from http-server component merge into lobby.
+
+## Overview
+
+This project is a server side component, a stand-alone http(s) server.
+
+### Functionality Provided / Strategic fit in the TripleA project
+
+We envision two kinds of client-server communication, over http or over websocket.
+Currently, lobby-server, uses Java RMI, which we would like to deprecate as it 
```

Comment:
This came up a lot of time.
Techincally we're not using RMI, but something very similar. It's probably good to explicitly state that here in the README